README.md

```diff
@@ -1,2 +1,36 @@
 # task-manager
 Create task manager backend service
+
+# Entitites
+
+# Task Table Schema
+
+All fields are NOT NULL
+
+Task_id  - integer  primary auto_increment
+
+Task_title - varchar(100)
+
+Task_description - varchar(200)
+
+Customer_id - int  foreign key
+
+Created_at - date
+
+updated_at   -date
+
+Task_Priority  -  int
+
```

Comment:
Task Table name = "task_info"
customer_id = Long
task_priority = Short

and Add is_archived = bool